apps/web-giddh/src/app/all-modules/all-modules.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="container-fluid pt-2 pb-2">
+    <div class="clearfix reports-main">
+        <div class="row">
+            <div class="col-md-3 col-sm-3 mb-5 col-xs-6">
+                <p>General</p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/contact/customer']">Customer</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/contact/aging-report']">Aging Report </a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/contact/vendor']">Vendor</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/accounting-voucher']">Journal Voucher</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/']">Create Account</a></p>
```

Comment:
Please check the flow of this path what happens when User clicks on this link as it will not go anywhere